content/en/tracing/profiler/search_profiles.md

```diff
@@ -158,7 +158,9 @@ Mutex
 : Shows the time functions have been waiting on mutexes during the profiling period (default: 60s). The stack traces in this profile point the `Unlock()` operation that allowed another goroutine blocked on the mutex to proceed. Short mutex contentions using spinlocks are not captured by this profile, but can be seen in the CPU profile. See also the note about how this measure changes in version `1.33.0` in [Delta profiles](#delta-profiles).
 
 Block
-: Shows the time functions have been waiting on mutexes and channel operations during the profiling period (default: 60s). Sleep, GC, Network and Syscall operations are not captured by this profile. Blocking operations are only captured after they become unblocked, so this profile cannot be used to debug applications that appear to be stuck. For mutex contentions, the stack traces in this profile point to blocked `Lock()` operations. This will tell you where your program is getting blocked, while the mutex profile tells you what part of your program is causing the contention. See our [Block Profiling in Go][1] research for more in-depth information. See also the note about how this measure changes in version `1.33.0` in [Delta profiles](#delta-profiles). *Note:* The block profiler can cause noticeable overhead for production workloads. If enabling it in production, prefer high rates (such as `100000000`, i.e. 100 milliseconds) and look for signs of increased latency or CPU utilization.
+: Shows the time functions have been waiting on mutexes and channel operations during the profiling period (default: 60s). Sleep, GC, Network and Syscall operations are not captured by this profile. Blocking operations are only captured after they become unblocked, so this profile cannot be used to debug applications that appear to be stuck. For mutex contentions, the stack traces in this profile point to blocked `Lock()` operations. This will tell you where your program is getting blocked, while the mutex profile tells you what part of your program is causing the contention. See our [Block Profiling in Go][1] research for more in-depth information. See also the note about how this measure changes in version `1.33.0` in [Delta profiles](#delta-profiles). 
+
+**Note:** The block profiler can cause noticeable overhead for production workloads. If enabling it in production, prefer high rates (such as `100000000`, that is 100 milliseconds) and look for signs of increased latency or CPU utilization.
```

Comment:
@maycmlee 👋 hey, we just noticed that this ended up breaking the box listing all the go profiles into two boxes which looks a bit odd. Was this intentional or should we send a PR to render the note text within the box?

![image](https://user-images.githubusercontent.com/15000/155555620-eb460cc6-b6fc-4838-8708-a74948f22c7f.png)

Reply:
Hey @felixge! Thanks for catching that! It's not intentional, I must have missed it in the preview. I'll open a PR to fix it.

Reply:
PR: https://github.com/DataDog/documentation/pull/13211

Reply:
Thank you so much for the fast fix : )